src/helpers/mentions.js

```diff
@@ -1,6 +1,7 @@
 /**
  * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later
+ * @param text
```

Comment:
Please add a sepaerate documentation block for the method, this one is just for the file copyright headers